meg_runtime/locking/lockFile.py

```diff
@@ -0,0 +1,116 @@
+"""MEG system lockfile parser
+
+Can be used to parse the lock file and preform operations on the lockfile
```

Comment:
Did you mean preform or perform? It could be either but I think you probably mean perform?